rpcserverhelp.go

```diff
@@ -156,8 +156,8 @@ var helpDescsEnUS = map[string]string{
 	"decodescriptresult-asm":       "Disassembly of the script",
 	"decodescriptresult-reqSigs":   "The number of required signatures",
 	"decodescriptresult-type":      "The type of the script (e.g. 'pubkeyhash')",
-	"decodescriptresult-addresses": "The decred addresses associated with this script",
-	"decodescriptresult-p2sh":      "The script hash for use in pay-to-script-hash transactions",
+	"decodescriptresult-addresses": "The bitcoin addresses associated with this script",
```

Comment:
shouldn't this remain decred?

Reply:
Good catch.  I glossed right over that.

Reply:
Good catch.  I glossed right over that while checking the merges were indeed NOOPs and everything is already applied.

Reply:
oops, correcting.